tests/src/FunctionalJavascript/WebformCivicrmTestBase.php

```diff
@@ -51,16 +51,38 @@ abstract class WebformCivicrmTestBase extends CiviCrmTestBase {
    */
   protected function setUp() {
     parent::setUp();
+
+    // Make sure we are using distinct default and administrative themes for
+    // the duration of these tests.
+    \Drupal::service('theme_installer')->install(['bartik', 'seven']);
+    $this->config('system.theme')
+      ->set('default', 'bartik')
+      ->set('admin', 'seven')
+      ->save();
+
```

Comment:
I'm going to put this into a separate PR. We have addressed the permission issue already in #478 🎉